src/tree/v2/providers/BuiltInApplicationResourceItem.ts

```diff
@@ -5,6 +5,8 @@ import { BuiltInResourceModelBase } from './BuiltInResourceModelBase';
 
 export class BuiltInApplicationResourceItem implements BuiltInResourceModelBase {
     constructor(private readonly resource: ApplicationResource) {
+        this.id = resource.id;
```

Comment:
Using Azure resource ID as tree item ID could bite us.

Reply:
I don't think I meant to include this change in this PR. This id is needed for the view properties command